<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationDialog</class>
 <widget class="QDialog" name="RegistrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>260</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="UsernameLine">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>56</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Введите свой логин</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PasswordLine">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>56</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Введите пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PasswordLine_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>56</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Введите пароль повторно</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RegisterButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>56</height>
       </size>
      </property>
      <property name="text">
       <string>Зарегестрироваться</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
